lib/quandl/connection.rb

```diff
@@ -5,7 +5,7 @@ def self.request(http_verb, url, options = {}, &block)
       headers = options.delete(:headers) || {}
       accept_value = 'application/json'
       accept_value += ", application/vnd.quandl+json;version=#{ApiConfig.api_version}" if ApiConfig.api_version
-      headers = { accept: accept_value }.merge(headers)
+      headers = { accept: accept_value, request_source: 'ruby', request_source_version: Quandl::VERSION }.merge(headers)
```

Comment:
Have your tried sending these to wikiposit and seeing what header it gets to ensure that we need to use `_` instead of `-`